[2021-12-28 13:48:21,253] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_kilkenny.scrap_data scheduled__2021-12-27T09:50:00+00:00 [queued]>
[2021-12-28 13:48:21,275] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_kilkenny.scrap_data scheduled__2021-12-27T09:50:00+00:00 [queued]>
[2021-12-28 13:48:21,276] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-28 13:48:21,277] {taskinstance.py:1242} INFO - Starting attempt 4 of 4
[2021-12-28 13:48:21,278] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-28 13:48:21,303] {taskinstance.py:1262} INFO - Executing <Task(SparkSubmitOperator): scrap_data> on 2021-12-27 09:50:00+00:00
[2021-12-28 13:48:21,311] {standard_task_runner.py:52} INFO - Started process 56046 to run task
[2021-12-28 13:48:21,320] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'scraping_daft_kilkenny', 'scrap_data', 'scheduled__2021-12-27T09:50:00+00:00', '--job-id', '5114', '--raw', '--subdir', 'DAGS_FOLDER/scraping_daft_kilkenny.py', '--cfg-path', '/tmp/tmp_jl68aeh', '--error-file', '/tmp/tmpoop9rxa3']
[2021-12-28 13:48:21,323] {standard_task_runner.py:77} INFO - Job 5114: Subtask scrap_data
[2021-12-28 13:48:21,544] {logging_mixin.py:109} INFO - Running <TaskInstance: scraping_daft_kilkenny.scrap_data scheduled__2021-12-27T09:50:00+00:00 [running]> on host 0c125a5bba95
[2021-12-28 13:48:21,887] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clement.liscoet@gmail.com
AIRFLOW_CTX_DAG_OWNER=clemoni
AIRFLOW_CTX_DAG_ID=scraping_daft_kilkenny
AIRFLOW_CTX_TASK_ID=scrap_data
AIRFLOW_CTX_EXECUTION_DATE=2021-12-27T09:50:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-27T09:50:00+00:00
[2021-12-28 13:48:21,943] {base.py:79} INFO - Using connection to: id: spark_default. Host: spark://spark:7077, Port: None, Schema: , Login: ***, Password: ***, extra: {}
[2021-12-28 13:48:21,950] {spark_submit.py:360} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py kilkenny
[2021-12-28 13:48:26,843] {spark_submit.py:514} INFO - Using properties file: null
[2021-12-28 13:48:27,124] {spark_submit.py:514} INFO - WARNING: An illegal reflective access operation has occurred
[2021-12-28 13:48:27,125] {spark_submit.py:514} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.6/site-packages/pyspark/jars/spark-unsafe_2.12-3.2.0.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2021-12-28 13:48:27,126] {spark_submit.py:514} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2021-12-28 13:48:27,127] {spark_submit.py:514} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2021-12-28 13:48:27,128] {spark_submit.py:514} INFO - WARNING: All illegal access operations will be denied in a future release
[2021-12-28 13:48:27,363] {spark_submit.py:514} INFO - Parsed arguments:
[2021-12-28 13:48:27,367] {spark_submit.py:514} INFO - master                  spark://spark:7077
[2021-12-28 13:48:27,368] {spark_submit.py:514} INFO - deployMode              null
[2021-12-28 13:48:27,370] {spark_submit.py:514} INFO - executorMemory          null
[2021-12-28 13:48:27,371] {spark_submit.py:514} INFO - executorCores           null
[2021-12-28 13:48:27,374] {spark_submit.py:514} INFO - totalExecutorCores      null
[2021-12-28 13:48:27,377] {spark_submit.py:514} INFO - propertiesFile          null
[2021-12-28 13:48:27,379] {spark_submit.py:514} INFO - driverMemory            null
[2021-12-28 13:48:27,381] {spark_submit.py:514} INFO - driverCores             null
[2021-12-28 13:48:27,382] {spark_submit.py:514} INFO - driverExtraClassPath    null
[2021-12-28 13:48:27,383] {spark_submit.py:514} INFO - driverExtraLibraryPath  null
[2021-12-28 13:48:27,386] {spark_submit.py:514} INFO - driverExtraJavaOptions  null
[2021-12-28 13:48:27,387] {spark_submit.py:514} INFO - supervise               false
[2021-12-28 13:48:27,389] {spark_submit.py:514} INFO - queue                   null
[2021-12-28 13:48:27,390] {spark_submit.py:514} INFO - numExecutors            null
[2021-12-28 13:48:27,391] {spark_submit.py:514} INFO - files                   null
[2021-12-28 13:48:27,392] {spark_submit.py:514} INFO - pyFiles                 null
[2021-12-28 13:48:27,394] {spark_submit.py:514} INFO - archives                null
[2021-12-28 13:48:27,395] {spark_submit.py:514} INFO - mainClass               null
[2021-12-28 13:48:27,396] {spark_submit.py:514} INFO - primaryResource         file:/opt/***/jobs/get_ads_per_county.py
[2021-12-28 13:48:27,398] {spark_submit.py:514} INFO - name                    scrap data
[2021-12-28 13:48:27,399] {spark_submit.py:514} INFO - childArgs               [kilkenny]
[2021-12-28 13:48:27,400] {spark_submit.py:514} INFO - jars                    null
[2021-12-28 13:48:27,401] {spark_submit.py:514} INFO - packages                null
[2021-12-28 13:48:27,403] {spark_submit.py:514} INFO - packagesExclusions      null
[2021-12-28 13:48:27,404] {spark_submit.py:514} INFO - repositories            null
[2021-12-28 13:48:27,405] {spark_submit.py:514} INFO - verbose                 true
[2021-12-28 13:48:27,406] {spark_submit.py:514} INFO - 
[2021-12-28 13:48:27,408] {spark_submit.py:514} INFO - Spark properties used, including those specified through
[2021-12-28 13:48:27,409] {spark_submit.py:514} INFO - --conf and those from the properties file null:
[2021-12-28 13:48:27,410] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-28 13:48:27,413] {spark_submit.py:514} INFO - 
[2021-12-28 13:48:27,415] {spark_submit.py:514} INFO - 
[2021-12-28 13:48:28,237] {spark_submit.py:514} INFO - Main class:
[2021-12-28 13:48:28,240] {spark_submit.py:514} INFO - org.apache.spark.deploy.PythonRunner
[2021-12-28 13:48:28,242] {spark_submit.py:514} INFO - Arguments:
[2021-12-28 13:48:28,244] {spark_submit.py:514} INFO - file:/opt/***/jobs/get_ads_per_county.py
[2021-12-28 13:48:28,245] {spark_submit.py:514} INFO - null
[2021-12-28 13:48:28,245] {spark_submit.py:514} INFO - kilkenny
[2021-12-28 13:48:28,247] {spark_submit.py:514} INFO - --verbose
[2021-12-28 13:48:28,251] {spark_submit.py:514} INFO - Spark config:
[2021-12-28 13:48:28,252] {spark_submit.py:514} INFO - (spark.app.name,scrap data)
[2021-12-28 13:48:28,253] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-28 13:48:28,253] {spark_submit.py:514} INFO - (spark.submit.pyFiles,)
[2021-12-28 13:48:28,274] {spark_submit.py:514} INFO - (spark.submit.deployMode,client)
[2021-12-28 13:48:28,275] {spark_submit.py:514} INFO - Classpath elements:
[2021-12-28 13:48:28,276] {spark_submit.py:514} INFO - 
[2021-12-28 13:48:28,277] {spark_submit.py:514} INFO - 
[2021-12-28 13:48:28,278] {spark_submit.py:514} INFO - 
[2021-12-28 13:48:30,420] {spark_submit.py:514} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2021-12-28 13:48:30,446] {spark_submit.py:514} INFO - 21/12/28 13:48:30 INFO SparkContext: Running Spark version 3.2.0
[2021-12-28 13:48:30,658] {spark_submit.py:514} INFO - 21/12/28 13:48:30 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2021-12-28 13:48:31,091] {spark_submit.py:514} INFO - 21/12/28 13:48:31 INFO ResourceUtils: ==============================================================
[2021-12-28 13:48:31,093] {spark_submit.py:514} INFO - 21/12/28 13:48:31 INFO ResourceUtils: No custom resources configured for spark.driver.
[2021-12-28 13:48:31,095] {spark_submit.py:514} INFO - 21/12/28 13:48:31 INFO ResourceUtils: ==============================================================
[2021-12-28 13:48:31,097] {spark_submit.py:514} INFO - 21/12/28 13:48:31 INFO SparkContext: Submitted application: Get Ads per county
[2021-12-28 13:48:31,167] {spark_submit.py:514} INFO - 21/12/28 13:48:31 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2021-12-28 13:48:31,192] {spark_submit.py:514} INFO - 21/12/28 13:48:31 INFO ResourceProfile: Limiting resource is cpu
[2021-12-28 13:48:31,193] {spark_submit.py:514} INFO - 21/12/28 13:48:31 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2021-12-28 13:48:31,405] {spark_submit.py:514} INFO - 21/12/28 13:48:31 INFO SecurityManager: Changing view acls to: ***
[2021-12-28 13:48:31,406] {spark_submit.py:514} INFO - 21/12/28 13:48:31 INFO SecurityManager: Changing modify acls to: ***
[2021-12-28 13:48:31,407] {spark_submit.py:514} INFO - 21/12/28 13:48:31 INFO SecurityManager: Changing view acls groups to:
[2021-12-28 13:48:31,409] {spark_submit.py:514} INFO - 21/12/28 13:48:31 INFO SecurityManager: Changing modify acls groups to:
[2021-12-28 13:48:31,411] {spark_submit.py:514} INFO - 21/12/28 13:48:31 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(***); groups with view permissions: Set(); users  with modify permissions: Set(***); groups with modify permissions: Set()
[2021-12-28 13:48:32,359] {spark_submit.py:514} INFO - 21/12/28 13:48:32 INFO Utils: Successfully started service 'sparkDriver' on port 44995.
[2021-12-28 13:48:32,540] {spark_submit.py:514} INFO - 21/12/28 13:48:32 INFO SparkEnv: Registering MapOutputTracker
[2021-12-28 13:48:32,660] {spark_submit.py:514} INFO - 21/12/28 13:48:32 INFO SparkEnv: Registering BlockManagerMaster
[2021-12-28 13:48:32,719] {spark_submit.py:514} INFO - 21/12/28 13:48:32 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2021-12-28 13:48:32,721] {spark_submit.py:514} INFO - 21/12/28 13:48:32 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2021-12-28 13:48:32,734] {spark_submit.py:514} INFO - 21/12/28 13:48:32 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2021-12-28 13:48:32,795] {spark_submit.py:514} INFO - 21/12/28 13:48:32 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-be75c60f-5b4a-4df6-b0b1-dd545b175427
[2021-12-28 13:48:32,849] {spark_submit.py:514} INFO - 21/12/28 13:48:32 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2021-12-28 13:48:32,929] {spark_submit.py:514} INFO - 21/12/28 13:48:32 INFO SparkEnv: Registering OutputCommitCoordinator
[2021-12-28 13:48:33,934] {spark_submit.py:514} INFO - 21/12/28 13:48:33 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2021-12-28 13:48:33,949] {spark_submit.py:514} INFO - 21/12/28 13:48:33 WARN Utils: Service 'SparkUI' could not bind on port 4041. Attempting port 4042.
[2021-12-28 13:48:33,954] {spark_submit.py:514} INFO - 21/12/28 13:48:33 WARN Utils: Service 'SparkUI' could not bind on port 4042. Attempting port 4043.
[2021-12-28 13:48:34,022] {spark_submit.py:514} INFO - 21/12/28 13:48:34 INFO Utils: Successfully started service 'SparkUI' on port 4043.
[2021-12-28 13:48:34,529] {spark_submit.py:514} INFO - 21/12/28 13:48:34 INFO SparkUI: Bound SparkUI to 0.0.0.0, and started at http://0c125a5bba95:4043
[2021-12-28 13:48:35,526] {spark_submit.py:514} INFO - 21/12/28 13:48:35 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-28 13:48:35,788] {spark_submit.py:514} INFO - 21/12/28 13:48:35 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-28 13:48:35,789] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-28 13:48:35,790] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-28 13:48:35,791] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-28 13:48:35,792] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-28 13:48:35,794] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-28 13:48:35,795] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-28 13:48:35,796] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-28 13:48:35,796] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-28 13:48:35,798] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-28 13:48:35,799] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-28 13:48:35,800] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:48:35,801] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-28 13:48:35,802] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-28 13:48:35,803] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-28 13:48:35,804] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-28 13:48:35,805] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-28 13:48:35,806] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-28 13:48:35,807] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-28 13:48:35,807] {spark_submit.py:514} INFO - ... 4 more
[2021-12-28 13:48:35,808] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-28 13:48:35,809] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-28 13:48:35,810] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-28 13:48:35,811] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-28 13:48:35,811] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-28 13:48:35,812] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-28 13:48:35,813] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-28 13:48:35,814] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-28 13:48:35,815] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-28 13:48:35,816] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-28 13:48:35,816] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-28 13:48:35,817] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-28 13:48:35,818] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-28 13:48:35,820] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-28 13:48:35,820] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-28 13:48:35,821] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-28 13:48:35,823] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-28 13:48:35,823] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-28 13:48:35,824] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-28 13:48:35,825] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-28 13:48:35,826] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-28 13:48:35,827] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-28 13:48:35,827] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-28 13:48:35,828] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-28 13:48:35,829] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-28 13:48:35,830] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-28 13:48:35,831] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-28 13:48:35,832] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-28 13:48:35,832] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-28 13:48:35,833] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-28 13:48:35,834] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-28 13:48:35,835] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-28 13:48:35,836] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-28 13:48:35,837] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-28 13:48:35,837] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-28 13:48:35,838] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-28 13:48:35,839] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-28 13:48:35,840] {spark_submit.py:514} INFO - ... 1 more
[2021-12-28 13:48:55,490] {spark_submit.py:514} INFO - 21/12/28 13:48:55 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-28 13:48:55,507] {spark_submit.py:514} INFO - 21/12/28 13:48:55 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-28 13:48:55,508] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-28 13:48:55,509] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-28 13:48:55,509] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-28 13:48:55,511] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-28 13:48:55,511] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-28 13:48:55,512] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-28 13:48:55,513] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-28 13:48:55,514] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-28 13:48:55,514] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-28 13:48:55,515] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-28 13:48:55,515] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:48:55,516] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-28 13:48:55,517] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-28 13:48:55,517] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-28 13:48:55,518] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-28 13:48:55,518] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-28 13:48:55,519] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-28 13:48:55,519] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-28 13:48:55,520] {spark_submit.py:514} INFO - ... 4 more
[2021-12-28 13:48:55,521] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-28 13:48:55,521] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-28 13:48:55,522] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-28 13:48:55,522] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-28 13:48:55,523] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-28 13:48:55,523] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-28 13:48:55,524] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-28 13:48:55,525] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-28 13:48:55,525] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-28 13:48:55,526] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-28 13:48:55,526] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-28 13:48:55,527] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-28 13:48:55,527] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-28 13:48:55,528] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-28 13:48:55,528] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-28 13:48:55,529] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-28 13:48:55,529] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-28 13:48:55,530] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-28 13:48:55,531] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-28 13:48:55,532] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-28 13:48:55,532] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-28 13:48:55,533] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-28 13:48:55,534] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-28 13:48:55,534] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-28 13:48:55,535] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-28 13:48:55,535] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-28 13:48:55,536] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-28 13:48:55,537] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-28 13:48:55,537] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-28 13:48:55,538] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-28 13:48:55,539] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-28 13:48:55,539] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-28 13:48:55,540] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-28 13:48:55,541] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-28 13:48:55,541] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-28 13:48:55,542] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-28 13:48:55,542] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-28 13:48:55,543] {spark_submit.py:514} INFO - ... 1 more
[2021-12-28 13:49:15,458] {spark_submit.py:514} INFO - 21/12/28 13:49:15 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-28 13:49:15,475] {spark_submit.py:514} INFO - 21/12/28 13:49:15 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-28 13:49:15,476] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-28 13:49:15,477] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-28 13:49:15,478] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-28 13:49:15,479] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-28 13:49:15,480] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-28 13:49:15,480] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-28 13:49:15,482] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-28 13:49:15,483] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-28 13:49:15,484] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-28 13:49:15,486] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-28 13:49:15,486] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:49:15,487] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-28 13:49:15,488] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-28 13:49:15,489] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-28 13:49:15,489] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-28 13:49:15,490] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-28 13:49:15,491] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-28 13:49:15,492] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-28 13:49:15,493] {spark_submit.py:514} INFO - ... 4 more
[2021-12-28 13:49:15,493] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-28 13:49:15,494] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-28 13:49:15,495] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-28 13:49:15,496] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-28 13:49:15,497] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-28 13:49:15,497] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-28 13:49:15,498] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-28 13:49:15,499] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-28 13:49:15,500] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-28 13:49:15,500] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-28 13:49:15,501] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-28 13:49:15,502] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-28 13:49:15,502] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-28 13:49:15,503] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-28 13:49:15,504] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-28 13:49:15,505] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-28 13:49:15,505] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-28 13:49:15,506] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-28 13:49:15,507] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-28 13:49:15,508] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-28 13:49:15,509] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-28 13:49:15,510] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-28 13:49:15,511] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-28 13:49:15,512] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-28 13:49:15,513] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-28 13:49:15,514] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-28 13:49:15,515] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-28 13:49:15,517] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-28 13:49:15,518] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-28 13:49:15,519] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-28 13:49:15,519] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-28 13:49:15,520] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-28 13:49:15,521] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-28 13:49:15,521] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-28 13:49:15,522] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-28 13:49:15,523] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-28 13:49:15,523] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-28 13:49:15,524] {spark_submit.py:514} INFO - ... 1 more
[2021-12-28 13:49:35,462] {spark_submit.py:514} INFO - 21/12/28 13:49:35 ERROR StandaloneSchedulerBackend: Application has been killed. Reason: All masters are unresponsive! Giving up.
[2021-12-28 13:49:35,465] {spark_submit.py:514} INFO - 21/12/28 13:49:35 WARN StandaloneSchedulerBackend: Application ID is not initialized yet.
[2021-12-28 13:49:35,512] {spark_submit.py:514} INFO - 21/12/28 13:49:35 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 43217.
[2021-12-28 13:49:35,514] {spark_submit.py:514} INFO - 21/12/28 13:49:35 INFO NettyBlockTransferService: Server created on 0c125a5bba95:43217
[2021-12-28 13:49:35,516] {spark_submit.py:514} INFO - 21/12/28 13:49:35 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2021-12-28 13:49:35,532] {spark_submit.py:514} INFO - 21/12/28 13:49:35 INFO SparkUI: Stopped Spark web UI at http://0c125a5bba95:4043
[2021-12-28 13:49:35,548] {spark_submit.py:514} INFO - 21/12/28 13:49:35 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 0c125a5bba95, 43217, None)
[2021-12-28 13:49:35,560] {spark_submit.py:514} INFO - 21/12/28 13:49:35 INFO BlockManagerMasterEndpoint: Registering block manager 0c125a5bba95:43217 with 434.4 MiB RAM, BlockManagerId(driver, 0c125a5bba95, 43217, None)
[2021-12-28 13:49:35,566] {spark_submit.py:514} INFO - 21/12/28 13:49:35 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 0c125a5bba95, 43217, None)
[2021-12-28 13:49:35,571] {spark_submit.py:514} INFO - 21/12/28 13:49:35 INFO StandaloneSchedulerBackend: Shutting down all executors
[2021-12-28 13:49:35,572] {spark_submit.py:514} INFO - 21/12/28 13:49:35 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 0c125a5bba95, 43217, None)
[2021-12-28 13:49:35,589] {spark_submit.py:514} INFO - 21/12/28 13:49:35 INFO CoarseGrainedSchedulerBackend$DriverEndpoint: Asking each executor to shut down
[2021-12-28 13:49:35,622] {spark_submit.py:514} INFO - 21/12/28 13:49:35 WARN StandaloneAppClient$ClientEndpoint: Drop UnregisterApplication(null) because has not yet connected to master
[2021-12-28 13:49:35,680] {spark_submit.py:514} INFO - 21/12/28 13:49:35 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2021-12-28 13:49:35,724] {spark_submit.py:514} INFO - 21/12/28 13:49:35 INFO MemoryStore: MemoryStore cleared
[2021-12-28 13:49:35,725] {spark_submit.py:514} INFO - 21/12/28 13:49:35 INFO BlockManager: BlockManager stopped
[2021-12-28 13:49:35,741] {spark_submit.py:514} INFO - 21/12/28 13:49:35 INFO BlockManagerMaster: BlockManagerMaster stopped
[2021-12-28 13:49:35,747] {spark_submit.py:514} INFO - 21/12/28 13:49:35 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2021-12-28 13:49:35,788] {spark_submit.py:514} INFO - 21/12/28 13:49:35 INFO SparkContext: Successfully stopped SparkContext
[2021-12-28 13:49:36,075] {spark_submit.py:514} INFO - 21/12/28 13:49:36 ERROR SparkContext: Error initializing SparkContext.
[2021-12-28 13:49:36,077] {spark_submit.py:514} INFO - java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2021-12-28 13:49:36,077] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2021-12-28 13:49:36,079] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2021-12-28 13:49:36,080] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2021-12-28 13:49:36,081] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2021-12-28 13:49:36,082] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2021-12-28 13:49:36,084] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2021-12-28 13:49:36,085] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2021-12-28 13:49:36,087] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2021-12-28 13:49:36,088] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2021-12-28 13:49:36,089] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2021-12-28 13:49:36,090] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2021-12-28 13:49:36,092] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2021-12-28 13:49:36,092] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2021-12-28 13:49:36,093] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2021-12-28 13:49:36,095] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2021-12-28 13:49:36,096] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:49:36,097] {spark_submit.py:514} INFO - 21/12/28 13:49:36 INFO SparkContext: SparkContext already stopped.
[2021-12-28 13:49:36,099] {spark_submit.py:514} INFO - Traceback (most recent call last):
[2021-12-28 13:49:36,100] {spark_submit.py:514} INFO - File "/opt/***/jobs/get_ads_per_county.py", line 13, in <module>
[2021-12-28 13:49:36,101] {spark_submit.py:514} INFO - .appName("Get Ads per county") \
[2021-12-28 13:49:36,102] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/session.py", line 228, in getOrCreate
[2021-12-28 13:49:36,105] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 392, in getOrCreate
[2021-12-28 13:49:36,107] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 147, in __init__
[2021-12-28 13:49:36,108] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 209, in _do_init
[2021-12-28 13:49:36,109] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 329, in _initialize_context
[2021-12-28 13:49:36,110] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/py4j-0.10.9.2-src.zip/py4j/java_gateway.py", line 1574, in __call__
[2021-12-28 13:49:36,111] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/py4j-0.10.9.2-src.zip/py4j/protocol.py", line 328, in get_return_value
[2021-12-28 13:49:36,112] {spark_submit.py:514} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling None.org.apache.spark.api.java.JavaSparkContext.
[2021-12-28 13:49:36,113] {spark_submit.py:514} INFO - : java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2021-12-28 13:49:36,114] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2021-12-28 13:49:36,115] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2021-12-28 13:49:36,117] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2021-12-28 13:49:36,118] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2021-12-28 13:49:36,119] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2021-12-28 13:49:36,120] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2021-12-28 13:49:36,121] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2021-12-28 13:49:36,122] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2021-12-28 13:49:36,123] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2021-12-28 13:49:36,124] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2021-12-28 13:49:36,125] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2021-12-28 13:49:36,126] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2021-12-28 13:49:36,127] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2021-12-28 13:49:36,128] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2021-12-28 13:49:36,129] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2021-12-28 13:49:36,129] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:49:36,130] {spark_submit.py:514} INFO - 
[2021-12-28 13:49:36,339] {spark_submit.py:514} INFO - 21/12/28 13:49:36 INFO ShutdownHookManager: Shutdown hook called
[2021-12-28 13:49:36,340] {spark_submit.py:514} INFO - 21/12/28 13:49:36 INFO ShutdownHookManager: Deleting directory /tmp/spark-6dbe6751-2a67-4c5c-b5ba-ab0c540d06d4
[2021-12-28 13:49:36,352] {spark_submit.py:514} INFO - 21/12/28 13:49:36 INFO ShutdownHookManager: Deleting directory /tmp/spark-002709a2-6c31-496b-9233-4ebe3d3e53f9
[2021-12-28 13:49:36,780] {taskinstance.py:1703} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py kilkenny. Error code is: 1.
[2021-12-28 13:49:36,795] {taskinstance.py:1280} INFO - Marking task as FAILED. dag_id=scraping_daft_kilkenny, task_id=scrap_data, execution_date=20211227T095000, start_date=20211228T134821, end_date=20211228T134936
[2021-12-28 13:49:36,869] {warnings.py:99} WARNING - /home/***/.local/lib/python3.6/site-packages/***/utils/email.py:102: PendingDeprecationWarning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
  send_mime_email(e_from=smtp_mail_from, e_to=recipients, mime_msg=msg, conn_id=conn_id, dryrun=dryrun)

[2021-12-28 13:49:36,870] {email.py:208} INFO - Email alerting: attempt 1
[2021-12-28 13:49:37,361] {email.py:220} INFO - Sent an alert email to ['clement.liscoet@gmail.com']
[2021-12-28 13:49:38,894] {standard_task_runner.py:91} ERROR - Failed to execute job 5114 for task scrap_data
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 292, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py kilkenny. Error code is: 1.
[2021-12-28 13:49:38,929] {local_task_job.py:154} INFO - Task exited with return code 1
[2021-12-28 13:49:39,066] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
